prep-html-links/contact-us/contact-us.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Contact Us</title>
+</head>
+<body>
+  <h1>Contact Us</h1>
+  <div><a href="index.html">Click here to go to the home page</a></div>
```

Comment:
Once the `index.html` file has been moved you will need to update the `href` of this anchor tag